qiita_pet/handlers/qiita_redbiom.py

```diff
@@ -49,13 +49,12 @@ def _redbiom_metadata_search(self, query, contexts):
             study_artifacts = defaultdict(lambda: defaultdict(list))
             for ctx in contexts:
                 # redbiom.fetch.data_from_samples returns a biom, which we
-                # will ignore, and a dict
+                # will ignore, and a dict: {sample_id_in_table: original_id}
```

Comment:
These changes shouldn't be here, right?

Reply:
Agree, this is separate from my changes.

Reply:
Thanks for confirming, probably related to the changelog.